timesketch/lib/tasks.py

```diff
@@ -1090,6 +1109,13 @@ def run_csv_jsonl(
         _set_datasource_status(timeline_id, file_path, "fail", error_message=str(e))
         raise
 
+    except errors.IndexNotReadyError as e:
+        # This triggers if the index does not return a good state.
+        logger.error("Unable to create index [%s]: %s", index_name, str(e))
```

Comment:
Do we want to prometheus this error to see how often this happens?

Reply:
Good point! Let me add that